dev-tests/report.html

```diff
@@ -0,0 +1,522 @@
+<!DOCTYPE html>
```

Comment:
Not sure if this was intentionally added?

Reply:
dang, keep accidentally getting this file after running pytests report. I'm gonna add it to .gitignore